proto/celestia/mint/v1/mint.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+package celestia.mint.v1;
+
+option go_package = "github.com/celestiaorg/celestia-app/x/mint/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/timestamp.proto";
+
+// Minter represents the mint state.
+message Minter {
+  // InflationRate is the rate at which new tokens should be minted for the
+  // current year. For example if InflationRate=0.1, then 10% of the total
+  // supply will be minted over the course of the year.
+  string inflation_rate = 1 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // AnnualProvisions is the total number of tokens to be minted in the current
+  // year due to inflation.
+  string annual_provisions = 2 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+
+  // GenesisTime is the timestamp of the genesis block.
+  google.protobuf.Timestamp genesis_time = 3 [(gogoproto.stdtime) = true];
```

Comment:
[TODO] @rootulp it's a bit confusing that this timestamp gets populated with the timestamp of block height = 1 and not genesis time. Explore clarifying this (either rename) or see how other chains name block 1

Reply:
nvm there won't be a block at height=0 per https://github.com/celestiaorg/networks/blob/309f961dcdc7e7353e8143cc59c1797e4c0fc326/blockspacerace/genesis.json#L4 so this variable name seems fine

Reply:
Good point https://github.com/celestiaorg/networks/blob/309f961dcdc7e7353e8143cc59c1797e4c0fc326/blockspacerace/genesis.json#L2 exists and it matches https://testnet.mintscan.io/celestia-incentivized-testnet/blocks/1 so I need to explore if it's accessible from this module